### ISSUE TITLE
[BUG - Release] Crashes when launching Little Nightmares II

### ISSUE THREAD

**Issue by acyc070** [UE4SS-RE/RE-UE4SS#694]
**Release**
Latest experimental release UE4SS 186 g8d38752

**Game and Engine Version**
Little Nightmares II 4.24

**Describe the bug**
When launching it shows "Fatal error!"

**Mods directory**
None(Default mods)

**To Reproduce**
1. Launch game
2. See Fatal error!
3. See NO ANY error in log

**Expected behavior**
The game launches without any error

**Desktop (please complete the following information):**
Windows 7

**Additional context**
I edited UE4SS-settings.ini and it doesn't help
Log:
[UE4SS.log](https://github.com/user-attachments/files/17495818/UE4SS.log)
I tested EXACTLY this release with other games and it WORKS so it's only game that crashes with UE4SS

**Comment by vr-commiter** [comment]
Hello, do you have any solution, I have encountered the same situation in other games.

**Comment by huanglvyuan** [comment]
Hello, do you have any solution, I have encountered the same situation in other games.

**Comment by acyc070** [comment]
I have a theory. It crashes because it uses custom plugins and that's broke offsets
![util](https://github.com/user-attachments/assets/e1df512a-fc71-4b0c-a38d-12e40f080e14)
![rwrew](https://github.com/user-attachments/assets/bc79fb0f-5110-4593-825c-cdb1e673be24)
![tools](https://github.com/user-attachments/assets/8d6ac9ea-0174-43c5-952c-3a4218364cca)

**Comment by huanglvyuan** [comment]
Sorry, I'm a novice. I don't understand what position you changed on the picture.

**Comment by acyc070** [comment]
I doesn't changed anything it's a developers custom plugins in uproject file

**Comment by acyc070** [comment]
I've experimented with uproject file and plugin manifest and here's results: those custom plugins are hard coded to be enabled and it probably needs to be fixed with new signatures

**Comment by InovadeGitHub** [comment]
yeah i also had this problem, fails with ln1 too. was disappointed cause i knew nothing (i still dont) about getting offsets

**Comment by acyc070** [comment]
> yeah i also had this problem, fails with ln1 too. was disappointed cause i knew nothing (i still dont) about getting offsets

First part is made on 4.13. It's VERY old version

**Comment by InovadeGitHub** [comment]
> > yeah i also had this problem, fails with ln1 too. was disappointed cause i knew nothing (i still dont) about getting offsets
> 
> First part is made on 4.13. It's VERY old version

yes. another thing is that they also made modifications to the engine for both games

**Comment by acyc070** [comment]
judging by first part log, it needs gmalloc

**Comment by Meep111** [comment]
it found Found GUObjectArray: 0x1431ccb08,it crashing on hooking right before Event loop start , this is a mod issue, i guess since rest is ok. might recheck mod hooking function in mod itself. bc u hook on event and when it starts it crashes.

**Comment by acyc070** [comment]
This can help but I can't test right now but I will try later: https://github.com/UE4SS-RE/RE-UE4SS/issues/693#issuecomment-2504101202

**Comment by acyc070** [comment]
> This can help but I can't test right now but I will try later: [#693 (comment)](https://github.com/UE4SS-RE/RE-UE4SS/issues/693#issuecomment-2504101202)

Nope. Didn't worked Little Nightmares II (Enhanced Editon) still crashing.

**Comment by LoopingMobiusStrip** [comment]
A fix has been posted to UEPseudo. Try building using this pull request. https://github.com/Re-UE4SS/UEPseudo/pull/108

**Comment by acyc070** [comment]
I actually don't want to build it by myself. Can you please provide builded program?

**Comment by igromanru** [comment]
> I actually don't want to build it by myself. Can you please provide builded program?

Just wait until the PR is merged and UE4SS is updated.

**Comment by acyc070** [comment]
What's about GUObjectArray for first part?

**Comment by acyc070** [comment]
It even couldn't find FText_Constructor.

**Comment by LoopingMobiusStrip** [comment]
The fix is not for Little Nightmares II (Enhanced Editon), the fix is for the standard Edition. My computer doesn't load the Enhanced Edition so I can't do anything about that.

**Comment by acyc070** [comment]
> The fix is not for Little Nightmares II (Enhanced Editon), the fix is for the standard Edition. My computer doesn't load the Enhanced Edition so I can't do anything about that.

Does it show any error?

**Comment by LoopingMobiusStrip** [comment]
idk it doesn't even load without UE4SS

**Comment by InovadeGitHub** [comment]
> I actually don't want to build it by myself. Can you please provide builded program?

[UE4SSNew.zip](https://github.com/user-attachments/files/18147741/UE4SSNew.zip) here

**Comment by acyc070** [comment]
> > I actually don't want to build it by myself. Can you please provide builded program?
> 
> [UE4SSNew.zip](https://github.com/user-attachments/files/18147741/UE4SSNew.zip) here

Thanks!

**Comment by acyc070** [comment]
But still we need to make UE4SS working with first part

**Comment by InovadeGitHub** [comment]
With the first Little Nightmares game?

**Comment by acyc070** [comment]
> With the first Little Nightmares game?

Yes.

**Comment by acyc070** [comment]
> It even couldn't find FText_Constructor.
> What's about GUObjectArray for first part?
UE4SS couldn't find these

**Comment by acyc070** [comment]
@InovadeGitHub I checked UE4SSNew with Enhanced Edition and it's still crashing! Even with bUseUObjectArrayCache = false, HookLoadMap = 0, HookInitGameState = 0 it's still crashing. The problem is different from 4.24 version

**Comment by SiND0L** [comment]
me i got this : [08:36:18.2412729] [Lua] ConsoleClass, GameViewport, or ViewportConsole is invalid

**Comment by acyc070** [comment]
> me i got this :
> 
> [08:40:08.3736111] [PS] Failed to find ConsoleManagerSingleton: found 2 unique values [7FF633D16E20, 7FF634B6EF80] [08:40:08.3736475] [PS] You can supply your own AOB in 'UE4SS_Signatures/ConsoleManager.lua' [08:36:18.2412729] [Lua] ConsoleClass, GameViewport, or ViewportConsole is invalid
> 
> and fatal error

'ConsoleManager' is not crash reason. It can be ignored.
